### PR TITLE
docs(website): add links to selenium-webdriver documentation

### PR DESCRIPTION
**Pull request by juliemr**
Add docs for common inherited functions on browser.
Add links to selenium-webdriver documentation.
Clean up the docgen code a little bit.
Make website tests pass again.

### CODE REVIEW

**Comment by juliemr** [comment]
cc @cnishina @andresdominguez

File: lib/selenium-webdriver/webdriver.js [hunk 1]
```diff
@@ -32,43 +32,129 @@ goog.require('webdriver.until');
...
 // //////////////////////////////////////////////////////////////////////////////
 // //
+// //  webdriver.WebDriver
+// //
+// //////////////////////////////////////////////////////////////////////////////
+/**
+ * Protractor's browser object is a wrapper for selenium-webdriver WebDriver.
+ * A full list of all functions available on WebDriver can be found
+ * in the selenium-webdriver
+ * <a href="http://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebDriver.html">documentation</a>
```
Comment (cnishina) [inline lib/selenium-webdriver/webdriver.js]: Nit: End sentence with a period.

File: lib/selenium-webdriver/webdriver.js [hunk 2]
```diff
@@ -32,43 +32,129 @@ goog.require('webdriver.until');
...
- *
- *     browser.findElement(By.id('not-there')).then(function(element) {
- *       alert('Found an element that was not expected to be there!');
- *     }, function(error) {
- *       alert('The element was not found, as expected');
- *     });
+ * Protractor's ElementFinders are wrappers for selenium-webdriver WebElement.
+ * A full list of all functions available on WebElement can be found
+ * in the selenium-webdriver
+ * <a href="http://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElement.html">documentation</a>
```
Comment (cnishina) [inline lib/selenium-webdriver/webdriver.js]: Nit: End sentence with a period.

**Comment by cnishina** [comment]
Other than nits. LGTM

File: lib/selenium-webdriver/webdriver.js [hunk 3]
```diff
@@ -32,43 +32,129 @@ goog.require('webdriver.until');
...
+ * @constructor
+ */
+webdriver.WebDriver = function() {};
+
+/**
+ * Creates a new action sequence using this driver. The sequence will not be
+ * scheduled for execution until {@link webdriver.ActionSequence#perform} is
+ * called. Example:
+ *
+ *     browser.actions().
```
Comment (andresdominguez) [inline lib/selenium-webdriver/webdriver.js]: Add @example here and below.

Fix indentation

File: lib/selenium-webdriver/webdriver.js (hunk 2 above)
Comment (juliemr) [inline lib/selenium-webdriver/webdriver.js]: done.

**Comment by juliemr** [comment]
Merged as 47b8310bae827bb032fac849939a7b9bfccc2367

File: lib/selenium-webdriver/webdriver.js (hunk 3 above)
Comment (juliemr) [inline lib/selenium-webdriver/webdriver.js]: Done with c6334888a8dd7633791c9830bca5a8a8acb3c816

**Pull request closed without merging**